library/src/main/java/com/bumptech/glide/request/RequestListener.java

```diff
@@ -50,14 +51,14 @@
    * }
    * }</pre>
    *
-   * @param e The maybe {@code null} exception containing information about why the request failed.
-   * @param model The model we were trying to load when the exception occurred.
-   * @param target The {@link Target} we were trying to load the image into.
+   * @param e               The maybe {@code null} exception containing information about why the request failed.
+   * @param model           The model we were trying to load when the exception occurred.
+   * @param target          The {@link Target} we were trying to load the image into.
    * @param isFirstResource {@code true} if this exception is for the first resource to load.
    * @return {@code true} to prevent {@link Target#onLoadFailed(Drawable)} from being called on
-   *     {@code target}, typically because the listener wants to update the {@code target} or the
-   *     object the {@code target} wraps itself or {@code false} to allow {@link
-   *     Target#onLoadFailed(Drawable)} to be called on {@code target}.
+   * {@code target}, typically because the listener wants to update the {@code target} or the
+   * object the {@code target} wraps itself or {@code false} to allow {@link
+   * Target#onLoadFailed(Drawable)} to be called on {@code target}.
```

Comment:
Please revert :)

Reply:
No idea how this space was added. I didn't even touch javadocs, and it would be weird for the formatter to add it.

Reply:
Look, I don't see that I've changed the Javadocs in the logs:

![image](https://user-images.githubusercontent.com/5357526/190845604-dd61ccae-51d3-430f-93b2-2cafcd832553.png)

I also don't see this space in code, so I don't understand where you got it from.